app/components/Admin/NaicsCodeSelectionComponent.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import React from 'react';
+import {ListGroup} from 'react-bootstrap';
+
+interface Props {
+  naicsCodes: {code: string; description: string; id: string}[];
+}
+
+export const NaicsCodeSelectionComponent: React.FunctionComponent<Props> = ({
+  naicsCodes
+}) => {
+  const router = useRouter();
+
+  return (
+    <>
+      <div className="scrollable" tabIndex={0}>
+        <ListGroup variant="flush">
+          {naicsCodes.map(({code, description, id}) => (
+            <Link
+              key={id}
+              prefetch={false}
+              href={{
+                pathname: router.pathname,
+                query: {
+                  ...router.query,
+                  naicsCodeId: id
+                }
+              }}
+            >
+              <ListGroup.Item action active={router.query?.naicsCodeId === id}>
+                {description}
+                <br />
+                <b>{code}</b>
+              </ListGroup.Item>
+            </Link>
+          ))}
+        </ListGroup>
+      </div>
+      <style jsx global>
```

Comment:
not all of the styles below need to be global, only `.list-group-item.active` (at least until we use a bootstrap theme)

Reply:
(you can have two `<style>` tags in a component - one global and one not 🤓)